Makefile.am

```diff
@@ -2,27 +2,38 @@ bin_PROGRAMS=	scrypt
 noinst_PROGRAMS=	tests/test_scrypt tests/valgrind/potential-memleaks
 dist_man_MANS=$(scrypt_man_MANS)
 
+# These files fulfill the crypto_scrypt() function, with the exception of
```

Comment:
I spent a minute agonizing over whether it should be "fulfill", or "provide", or "implements", or something else.  I think the meaning is clear in context (particularly if you look at `test_scrypt`), but I also think that "fulfill" isn't quite the right word.